Doc/using/cmdline.rst

```diff
@@ -994,3 +1000,110 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
```

Comment:
What do you think of adding a new file to the doc? This section doesn't directly belong to the "Command line and environment". I suggest adding the new file to the Doc/library/ directory.

Example: ``Doc/library/sys_path_init.rst``.

IMO listing this file in https://docs.python.org/dev/library/modules.html is a good place.

Reply:
You are right, this section doesn't fit neatly into the "Command line and environment" page. I'm open to the idea of a new page. Be interested to know what @zooba  and @ned-deily think?

Reply:
I have moved the section into a new doc. Lets see what people think of this approach.